tools/install-monorepo.sh

```diff
@@ -18,102 +19,100 @@ if [[ $1 ]]; then
 	if ! command -v apt &>/dev/null; then
-		abort "Installer script requires 'apt' to ensure essentials are installed."
+		die "Installer script requires 'apt' to ensure essentials are installed."
 	fi
 	sudo apt update
 	sudo apt install build-essential
 elif [[ "$OS" != "Darwin" ]]; then
-	abort "Installer script is only supported on macOS and Linux."
+	die "Installer script is only supported on macOS and Linux."
 fi
 
 # Check for curl and git
 if ! command -v git &>/dev/null; then
-	abort "Installer script requires 'git' to be installed."
+	die "Installer script requires 'git' to be installed."
 fi
 
 if ! command -v curl &>/dev/null; then
-	abort "Installer script requires 'curl' to be installed"
+	die "Installer script requires 'curl' to be installed"
 fi
 
 # Check of Homebrew and nvm are installed
-echo "Checking if Homebrew is installed..."
+info "Checking if Homebrew is installed..."
 if ! command -v brew &>/dev/null; then
-	echo "Installing Homebrew"
+	info "Installing Homebrew"
 	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 	if [[ -n "$ON_LINUX" ]]; then # Add homebrew to PATH
 		echo 'eval "$("/home/linuxbrew/.linuxbrew/bin/brew" shellenv)"' >> "$HOME/.profile"
 		eval "$("/home/linuxbrew/.linuxbrew/bin/brew" shellenv)"
 		PATH="/home/linuxbrew/.linuxbrew/bin:$PATH"
 	fi
 else
-	echo "Updating brew"
+	info "Updating brew"
 	brew update
 	# Brew can be finicky on MacOS
 	if [[ $? -ne 0 ]]; then
 		 echo "Reinstalling Homebrew"
-   	 	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+		 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 	fi
 fi
 
-echo "Checking if NVM is installed..."
+info "Checking if NVM is installed..."
 if ! command -v nvm &>/dev/null; then
-	echo "Installing nvm"
+	info "Installing nvm"
 	curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash && export NVM_DIR=$HOME/.nvm && source $NVM_DIR/nvm.sh  --no-use
 else
-	echo "Updating nvm"
+	info "Updating nvm"
 	curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
 fi
 
 # Install and use the correct version of Node.js
-echo "Installing Node.js"
+info "Installing Node.js"
 nvm install && nvm use
 
 # Install our requirements
-echo "Checking Bash version..."
+info "Checking Bash version..."
 if [[ -z "${BASH_VERSINFO}" || -z "${BASH_VERSINFO[0]}" || ${BASH_VERSINFO[0]} -lt 4 ]]; then
 	brew install bash
 fi
 
-echo "Checking if jq is installed..."
+info "Checking if jq is installed..."
 if ! command -v jq &>/dev/null; then
-	echo "Installing jq"
+	info "Installing jq"
 	brew install jq
 fi
 
-echo "Checking if pnpm is installed..."
+info "Checking if pnpm is installed..."
 if ! command -v pnpm &>/dev/null; then
-	echo "Installing pnpm"
-	curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm
+	info "Installing pnpm"
+	curl -fsSL https://get.pnpm.io/install.sh | sh -
```

Comment:
```suggestion
	curl -fsSL https://get.pnpm.io/install.sh | sh -
	source ~/.zshrc
```

Adding this here resolved the issue for me.

Reply:
This will not work for people using bash instead of zsh. Or dash or fish or ksh.

For that matter it might not always work for people running zsh, if their `~/.zshrc` includes stuff that's not compatible with the bash that the current script is running with.

... So many different ways to install JS crap, and they all suck.

Reply:
Would this work?
```
if [[ $0 == "zsh" ]]; then
  source ~/.zshrc
fi
...
```

Reply:
It wouldn't solve all your problems, but personally this PR is an improvement for me and I'd like to get it merged :)

Reply:
No. But c766a4f might.